iroh-sync/src/store.rs

```diff
@@ -175,6 +181,23 @@ pub trait Store: std::fmt::Debug + Clone + Send + Sync + 'static {
 
     /// Get peers to use for syncing a document.
     fn get_sync_peers(&self, namespace: &NamespaceId) -> Result<Option<Self::PeersIter<'_>>>;
+
+    /// Set the download policy for a document.
+    fn set_download_policy(&self, namespace: &NamespaceId, policy: DownloadPolicy) -> Result<()>;
+    /// Get the download policy for a document.
+    fn get_download_policy(&self, namespace: &NamespaceId) -> Result<DownloadPolicy>;
```

Comment:
why is this duplicated from the `DownloadPolicyStore` trait?

Reply:
That's a good point. I guess both can be moved completely to the alt trait